Python/Python.sublime-syntax

```diff
@@ -14,6 +14,7 @@ file_extensions:
   - gyp
   - gypi
   - Snakefile
+  - wscript
```

Comment:
and these files should be picked up as Python by the `first_line_match` anyway: https://waf.io/book/#_declaring_waf_commands